django/db/backends/base/base.py

```diff
@@ -243,9 +243,13 @@ def _close(self):
 
     # ##### Generic wrappers for PEP-249 connection methods #####
 
-    def cursor(self):
-        """Create a cursor, opening a connection if necessary."""
-        return self._cursor()
+    def cursor(self, *args, **kwargs):
+        """
+        Create a cursor, opening a connection if necessary.
+        args and kwargs will be passed unchanged to underlying connection's
+        cursor.
```

Comment:
could you comment on the module not found error? should I use try except like before?